framework/common/servicedef/services_test.xml

```diff
@@ -47,6 +47,13 @@ under the License.
     <service name="testError" engine="java" export="true" validate="false" require-new-transaction="true" max-retry="1"
             location="org.apache.ofbiz.common.CommonServices" invoke="returnErrorService">
     </service>
+    <service name="test-clojure-svc" engine="clojure" export="true" validate="false" require-new-transaction="true"
```

Comment:
Same as previous :) : 'testClojureSvc'

Reply:
If it's not a hard rule I would prefer to keep them as is.

Reply:
Done.